flexget/__init__.py

```diff
@@ -1,12 +1,21 @@
 #!/usr/bin/python
 import os
 import sys
+import threading
+from time import sleep
+
+from loguru import logger
 
 # __version__ import need to be first in order to avoid circular import within logger
 from ._version import __version__  # noqa
 from flexget import log
+from flexget.event import event
+from flexget.tray_icon import tray_icon
 from flexget.manager import Manager
 
+logger = logger.bind(name='main')
+manager_loaded = False
```

Comment:
This should probably be a `threading.Event`

Reply:
thanks, my threading game is weak. Event indeed seems perfect here